compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -1998,7 +2000,12 @@ public void visit(BLangInvocation invocationExpr) {
         }
 
         if (invocationExpr.actionInvocation || invocationExpr.async) {
```

Comment:
Is it correct to do this for all action invocations too? They may or may not be async right?

Reply:
I'll recheck and see.

Reply:
Fixed